secret-sender.rb

```diff
@@ -0,0 +1,25 @@
+require 'fileutils'
+
+class SecretSender < Formula
+  desc 'SecretSender is a little utility to securely share secrets to other users.'
+  homepage 'https://github.com/Shopify/secret-sender'
+  url 'https://github.com/Shopify/secret-sender/archive/v1.0.0.tar.gz'
+  sha256 'f395b5da63d6c031b756622b6ad8430e8ca54d60d8d7266d7132b1265c4e2d39'
+
+  depends_on 'go' => :build
+
+  bottle do
+    root_url "https://github.com/Shopify/secret-sender/releases/download/v1.0.0"
+    cellar :any_skip_relocation
+    sha256 "d99c4766085be488aff1f00e4c07af0690e1817fde68a4f660e496f60dc1a12d" => :high_sierra
```

Comment:
```
$ brew install https://raw.githubusercontent.com/Shopify/homebrew-shopify/183861ce1cca3ba307ad2ae7bb1dde8603ca71b7/secret-sender.rb
######################################################################## 100.0%
==> Downloading https://github.com/Shopify/secret-sender/releases/download/v1.0.0/secret-sender-1.0.0.high_sierra.bottle.tar.gz
Already downloaded: /Users/seb/Library/Caches/Homebrew/secret-sender-1.0.0.high_sierra.bottle.tar.gz
Error: SHA256 mismatch
Expected: d99c4766085be488aff1f00e4c07af0690e1817fde68a4f660e496f60dc1a12d
Actual: 31bc9093d41287d7147c444f7a42c2ab59c5f9643fdfa27281261bce2b548e19
Archive: /Users/seb/Library/Caches/Homebrew/secret-sender-1.0.0.high_sierra.bottle.tar.gz
To retry an incomplete download, remove the file above.
Warning: Bottle installation failed: building from source.
==> Downloading https://github.com/Shopify/secret-sender/archive/v1.0.0.tar.gz
Already downloaded: /Users/seb/Library/Caches/Homebrew/secret-sender-1.0.0.tar.gz
==> go build -o secret-sender github.com/Shopify/secret-sender
🍺  /usr/local/Cellar/secret-sender/1.0.0: 4 files, 2.8MB, built in 2 seconds
```

Reply:
This is because you had previously downloaded the one from S3. If you remove that file, it will work.

Reply:
Ran `brew cleanup -s` and tried again:
```
$ brew install https://raw.githubusercontent.com/Shopify/homebrew-shopify/183861ce1cca3ba307ad2ae7bb1dde8603ca71b7/secret-sender.rb
######################################################################## 100.0%
==> Downloading https://github.com/Shopify/secret-sender/releases/download/v1.0.0/secret-sender-1.0.0.high_sierra.bottle.tar.gz
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/135347800/fd866b26-6414-11e8-98e0-8bd8dac454c6?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20180531%2Fus-east-1%2Fs3%2Faws4_
######################################################################## 100.0%
==> Pouring secret-sender-1.0.0.high_sierra.bottle.tar.gz
🍺  /usr/local/Cellar/secret-sender/1.0.0: 4 files, 2.8MB
```
LGTM